module-presentation/src/main/java/com/depromeet/image/facade/ImageFacade.java

```diff
@@ -99,4 +101,11 @@ public List<ImageResponse> findImagesByMemoryId(Long memoryId) {
     public void deleteAllImagesByMemoryId(Long memoryId) {
         imageDeleteUseCase.deleteAllImagesByMemoryId(memoryId);
     }
+
+    private static boolean isDefaultProfileNumber(String imageName) {
+        return imageName.equals("1")
+                || imageName.equals("2")
+                || imageName.equals("3")
+                || imageName.equals("4");
```

Comment:
image 이름을 1,2,3,4로 하기보단 의미를 명확히 하기 위해 리스트 상수를 하나 만드는게 어떨까요?

```suggestion
                private static List<String> DEFAULT_IMAGE_NAMES = List.of("1", "2", "3", "4");
                .....
                
                private static boolean isDefaultProfileNumber(String imageName) {
                    return DEFAULT_IMAGE_NAMES.contains(imageName);
                }
```

Reply:
좋은 것 같습니다!